test/workers/test_websocket_worker.py

```diff
@@ -54,4 +57,8 @@ def test_websocket_workers_search(hook, start_proc):
     assert results[0].owner.id == "me"
     assert results[0].location.id == "fed2"
 
+    client_worker.ws.shutdown()
+    client_worker.ws.close()
+    time.sleep(0.1)
     server.terminate()
+    server.join()
```

Comment:
what does this join do?

Reply:
Removed. Leftovers of attempts to make the test pass